egs2/wsj0-2mix/enh1/run.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+min_or_max=min
+sample_rate=8k
+
+
+train_set="tr_${min_or_max}_${sample_rate}"
+dev_set="cv_${min_or_max}_${sample_rate}"
+eval_sets="tt_${min_or_max}_${sample_rate} "
+
+./enh.sh \
+    --train_set "${train_set}" \
+    --dev_set "${dev_set}" \
+    --eval_sets "${eval_sets}" \
+    --fs $sample_rate \
```

Comment:
why there are two sample_rate?

Reply:
Hmm, one is for `enh.sh`, the another one is passed to `data.sh` by `local_data_opts`. It seems not a good method to directly pass `sample_rate` to `data.sh` in `enh.sh`.

Reply:
I see. This is no problem to keep it. It's just a note.